frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue

```diff
@@ -26,11 +26,12 @@ const buttonList = computed(() => {
       })
     }
   })
+  // eslint-disable-next-line vue/no-side-effects-in-computed-properties
```

Comment:
Will you merge soon?

The longer we wait for your update of Nuxt, the more conflicts and warnings like this are accumulating, and I am afraid that the probability of introducing bugs when solving conflicts and warnings is growing.

Regarding this specific warning, I suppose that it is because the code in computed updates a global variable.
I made a PR to fix it. After the merge you will update from staging and the warning should disappear.

Reply:
nice thx